addons/mail/static/tests/owl/form_renderer_chatter_tests.js

```diff
@@ -0,0 +1,288 @@
+odoo.define('mail.FormRendererChatterTests', function (require) {
+"use strict";
+
+const FormView = require('web.FormView');
+
+const {
+    afterNextRender,
+    pause,
+    start,
+} = require('mail.messagingTestUtils');
+
+QUnit.module('mail.messaging', {}, function () {
+QUnit.module('Chatter', {
+    beforeEach: function () {
+        this.underscoreDebounce = _.debounce;
+        this.underscoreThrottle = _.throttle;
+        _.debounce = _.identity;
+        _.throttle = _.identity;
+        this.data = {
+            'ir.attachment': {
+                fields: {
+                    name: { type: 'char', string: "attachment name", required: true },
+                    res_model: { type: 'char', string: "res model" },
+                    res_id: { type: 'integer', string: "res id" },
+                    url: { type: 'char', string: 'url' },
+                    type: { type: 'selection', selection: [['url', "URL"], ['binary', "BINARY"]] },
+                    mimetype: { type: 'char', string: "mimetype" },
+                },
+                records: [
+                    {
+                        id: 1, type: 'url', mimetype: 'image/png', name: 'filename.jpg',
+                        res_id: 7, res_model: 'partner'
+                    },
+                    {
+                        id: 2, type: 'binary', mimetype: "application/x-msdos-program",
+                        name: "file2.txt", res_id: 7, res_model: 'partner'
+                    },
+                    {
+                        id: 3, type: 'binary', mimetype: "application/x-msdos-program",
+                        name: "file3.txt", res_id: 5, res_model: 'partner'
+                    },
```

Comment:
Not a fond of global data definition.
I prefer each test explicitly defining records they use, even if it means some duplication among tests.